docs/explanation/generic-rules.md

```diff
@@ -0,0 +1,68 @@
+# Generic alert rule groups
+The Canonical Observability Stack (COS) includes Generic alert rules which provide a minimal set of rules to inform admins when hosts in a deployment are unhealthy, unreachable, or otherwise unresponsive. This helps relieves charm authors from having to implement their host-health-related alerts per charm. 
```

Comment:
```suggestion
The Canonical Observability Stack (COS) includes Generic alert rules which provide a minimal set of rules to inform admins when hosts in a deployment are unhealthy, unreachable, or otherwise unresponsive. This helps relieve charm authors from having to implement their host-health-related alerts per charm. 
```
Oh oops that actually came from me :blush: